sdk/loadtestservice/Azure.Developer.LoadTesting/src/Azure.Developer.LoadTesting.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.Text.Json" VersionOverride="6.0.6"/>
```

Comment:
Why do we need to specify the json version here? We may not set version in project file.

Reply:
Please ref to lines: https://github.com/Azure/azure-sdk-for-net/pull/32268/files#diff-2b660017e47f166f6eaa396be0355b9d7a9aeb61f333a028a71d4c4fa4101708R361-R367

Reply:
We have changed the way, we were implementing LRO, hence, also removing dependency from this version.